src/keyboard/getEventProps.ts

```diff
@@ -0,0 +1,27 @@
+import {keyboardKey, keyboardState} from './types'
+
+export function getKeyEventProps(keyDef: keyboardKey, state: keyboardState) {
+  return {
+    key: keyDef.key,
+    code: keyDef.code,
+    altKey: state.modifiers.alt,
+    ctrlKey: state.modifiers.ctrl,
+    metaKey: state.modifiers.meta,
+    shiftKey: state.modifiers.shift,
+
+    // deprecated
```

Comment:
If we use the `@deprecated` annotation then vscode/TypeScript will offer some suggestions :)

```suggestion
    /** @deprecated use code instead */
```

https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAcwKZQNKoJ4FEBuqYUACgE5wAOAzgBQCUiA3gLABQiiZ6IZSrHTogDWOAFyIA5KOwARVMAB0MyQBp2QxBDgATVBOk55S7XrUahAQwA2mcVOpRLUVIoC2umMBioy1RTZQ5oKc0GTWWNgGjs6uHjpePn6KYdbBmm7olpHRTi7unt6+-plO6ULUABZedlEOeXGFSf5VNcEWnAD0AFTdiAACepTcELE6iCDUqFq60zBgjqiW492dHSI4AMKzBtWSFgC+7Eds7GenbNoLUFyo1IgAvCjokQREpBQ0DOfc-jLbeiAA

![image](https://user-images.githubusercontent.com/1500684/111334435-2bbd0d80-8639-11eb-9f0c-71cd06cda840.png)